src/main_operations/modules/apriltags/pre_processing/ai_acceleration/training/train_position_model.py

```
@@ -207,40 +215,69 @@ def build_grid_targets(
+        self.decay = decay
+        self.ema.requires_grad_(False)
+
+    def _clone_model(self, model: nn.Module) -> nn.Module:
+        model_copy = copy.deepcopy(model)
+        for p in model_copy.parameters():
+            p.requires_grad = False
+        return model_copy
+
+    @torch.no_grad()
+    def update(self, model: nn.Module) -> None:
+        for (name, ema_p), p in zip(
```

Ruff (0.14.0)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)